src/Symfony/Component/HttpKernel/BootingKernel.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel;
+
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\Exception\BadMethodCallException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Bundle\BootingBundle;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class BootingKernel implements KernelInterface
```

Comment:
No, it makes no sense. the decorated kernel can be a `TerminableInterface` instance, but we'll never call `terminate(Request $request, Response $response)` from a booting kernel. It's read-only, and moreover cannot finish a request as it can't even handle one ! 😅